### DISCUSSION TITLE
Deserializing JSON into AdmissionReview Object

### DISCUSSION THREAD

**Discussion by rnemeth90** [kubernetes-client/csharp#1249] [Q&A]
I am creating an admission controller to handle tainting new nodes in AKS. The "k8s.io/api/admission/v1beta1" for go defines a v1beta1.AdmissionReview struct. Does this c# library have a similar class? I'm not seeing anything in the documentation or examples, and I'm not sure what type I should deserialize the admission review into. I have a .NET Core web api with a simple controller action expecting a POST request:

```
        [HttpPost]
        [Route("node/mutate")]
        public async Task<IActionResult> Mutate([FromBody]object admissionRequest)
        {
            // what can I deserialize admissionRequest into? 

            return Json(admissionRequest);
        }
```

**Comment by tg123** [comment]
not included but you can use `KubernetesJson` or Standard json to deserialize it
